google-cloud-datastore/src/main/java/com/google/cloud/datastore/GqlQuery.java

```diff
@@ -282,9 +282,9 @@ static Binding fromPb(com.google.datastore.v1.GqlQueryParameter argPb) {
      * @param value a {@link FullEntity} object or a list of {@link FullEntity} objects that binds
      *     to a given name
      */
+    @Deprecated
     public Builder<V> setBinding(String name, FullEntity<?>... value) {
-      namedBindings.put(name, toBinding(EntityValue.MARSHALLER, Arrays.asList(value)));
-      return this;
+      throw new DatastoreException(12, "Binding entities is not supported.", "UNIMPLEMENTED");
```

Comment:
Why 12? Is there a constant somewhere to say what this means?

Reply:
This is from https://cloud.google.com/datastore/docs/concepts/errors, the link to canonical error code (https://github.com/google/googleapis/blob/master/google/rpc/code.proto) says the code for unimplemented is assigned 12 (but I am not sure they are the same think after a second look now)

Reply:
Oh I see. The codes are defined in a few places but should be consistent. Consider using [Status.Code](https://grpc.io/grpc-java/javadoc/io/grpc/Status.Code.html).

I think you can do
```java
Status.Code.UNIMPLEMENTED.value()
```